app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ApplicationRecord
+  has_many :works
```

Comment:
No validations?  Username can be blank?  What about requiring unique usernames?

This isn't essential, but something to think about.